src/main/java/Dice.java

```diff
@@ -0,0 +1,20 @@
+import java.util.*;
+
+public class Dice {
+
+    private Random random;
+
+    public Dice(Random random) {
+        this.random = random;
+    }
+
+    public int roll_dice() {
```

Comment:
Can we change this function name to `rollDice()`?

Reply:
Sure! I'll change it to `rollDice()`.